src/Microsoft.SqlTools.ServiceLayer/Utility/SqlScriptFormatters/FromSqlScript.cs

```diff
@@ -0,0 +1,148 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.Utility.SqlScriptFormatters
+{
+    /// <summary>
+    /// Provides utilities for converting from SQL script syntax into POCOs.
+    /// </summary>
+    public static class FromSqlScript
+    {
+        private static readonly Regex StringRegex = new Regex("^N?'(.*)'$", RegexOptions.Compiled);
```

Comment:
Minor: always comment on the regex in case we add to it later. In this case it's to `Detect String Literals inside error messages`?